### ISSUE TITLE
Langium

### ISSUE THREAD

**Issue by emilkrebs** [simple-icons/simple-icons#9512]
### Brand Name

Langium

### Website

https://langium.org/

### Popularity Metric

Global Rank: 2,012,149
GitHub stars: 470

### Official Resources for Icon and Color

https://github.com/eclipse-langium/langium-website/blob/main/hugo/static/assets/nib.svg

### Additional Comments

I am working for Langium and we are excited to see the icon.

### Contributing

Yes

**Comment by adamrusted** [comment]
Under discussion on [Discord](https://discord.com/channels/1142044630909726760/1149730177321029773).

**Comment by adamrusted** [comment]
Do we have any other brands currently in the project that we could run a [Google Trends](https://trends.google.com/trends/explore?date=all&q=Langium) comparison against, @emilkrebs?

**Comment by emilkrebs** [comment]
Maybe xtext?

**Comment by adamrusted** [comment]
I don't think we have either in the project? No stress. If we compare against those and check _their_ SimilarWeb and GitHub ranks, we might be able to work it out from there.

**Comment by emilkrebs** [comment]
Okay, the Google trends for `JetBrains MPS`, `MontiCore`, `xtext`, and `Langium` since the release of Langium 1.0 (16.12.2022) all seem to have the same popularity. Xtext is the most popular of all of them.
